sqlglot/dialects/redshift.py

```diff
@@ -47,6 +47,11 @@ class Redshift(Postgres):
     }
 
     class Parser(Postgres.Parser):
+        ID_VAR_TOKENS = {
+            *Postgres.Parser.ID_VAR_TOKENS,
+            TokenType.VALUES,
```

Comment:
does bigquery also define this?

Reply:
Yep, fyi I didn't move this logic into the metaclass because it's just these two that don't support `VALUES` and it felt overkill to dry it up like that. Maybe a third one would justify it.

Reply:
nah it's fine